src/phoenix/server/api/input_types/PatchDatasetExamplesInput.py

```diff
@@ -0,0 +1,35 @@
+from typing import List, Optional
+
+import strawberry
+from strawberry import UNSET
+from strawberry.relay import GlobalID
+from strawberry.scalars import JSON
+
+
+@strawberry.input
+class DatasetExamplePatch:
+    """
+    Contains the information needed to apply a patch revision to a dataset example.
+    """
+
+    example_id: GlobalID
+    input: Optional[JSON] = UNSET
+    output: Optional[JSON] = UNSET
```

Comment:
Should we explicitly forbid the deletion of input and output, i.e. returning an error in that case, instead of just ignoring it?

Reply:
Not sure if this addresses the concern, but if `input` or `output` is `UNSET`, nothing will be deleted and the `input` and `output` from the previous revision will be used.

Reply:
i was referring to whether we should raise error if `{}` is passed in for input or output, because right now it's silently ignored.

Reply:
Good catch. I think we can allow `{}` without harm. Addressed in 07f6430ace5273f2d728d839c4e2f97834126640
.